### PR TITLE
add space_to_depth and depth_to_space tests with NCHW data format

### PR DESCRIPTION
**Pull request by lei-Qiao**

### CODE REVIEW

File: tests/test_backend.py [hunk 1]
```diff
@@ -1482,6 +1482,23 @@ def test_space_to_depth(self):
...
         self._test_reorganize_data(tf.depth_to_space, [1, 1, 1, 4])
 
+    def _test_reorganize_data_gpu(self, op, shape):
+        x_val = make_xval(shape)
+        x = tf.placeholder(tf.float32, x_val.shape, name=_TFINPUT)
+        x_ = op(x, block_size=2, data_format="NCHW")
+        _ = tf.identity(x_, name=_TFOUTPUT)
+        self._run_test_case([_OUTPUT], {_INPUT: x_val})
+
+    @skip_tf_cpu("only tf_gpu can run maxpool with NCHW format")
```
Comment (zhijxu-MS) [inline tests/test_backend.py]: change maxpool to  Space2Depth.

File: tests/test_backend.py [hunk 2]
```diff
@@ -1482,6 +1482,23 @@ def test_space_to_depth(self):
...
+    @skip_caffe2_backend("Space2Depth not implemented")
+    def test_space_to_depth_gpu(self):
+        self._test_reorganize_data_gpu(tf.space_to_depth, [1, 1, 2, 2])
+
+    @skip_tf_cpu("only tf_gpu can run maxpool with NCHW format")
+    @skip_caffe2_backend("Space2Depth not implemented")
```
Comment (zhijxu-MS) [inline tests/test_backend.py]: Space2Depth  to depth_to_space

File: tests/test_backend.py [hunk 3]
```diff
@@ -1482,6 +1482,23 @@ def test_space_to_depth(self):
...
+        x_val = make_xval(shape)
+        x = tf.placeholder(tf.float32, x_val.shape, name=_TFINPUT)
+        x_ = op(x, block_size=2, data_format="NCHW")
+        _ = tf.identity(x_, name=_TFOUTPUT)
+        self._run_test_case([_OUTPUT], {_INPUT: x_val})
+
+    @skip_tf_cpu("only tf_gpu can run maxpool with NCHW format")
+    @skip_caffe2_backend("Space2Depth not implemented")
+    def test_space_to_depth_gpu(self):
+        self._test_reorganize_data_gpu(tf.space_to_depth, [1, 1, 2, 2])
```
Comment (zhijxu-MS) [inline tests/test_backend.py]: add test  case where change is not 1

File: tests/test_backend.py [hunk 4]
```diff
@@ -1482,6 +1482,23 @@ def test_space_to_depth(self):
...
+
+    @skip_tf_cpu("only tf_gpu can run maxpool with NCHW format")
+    @skip_caffe2_backend("Space2Depth not implemented")
+    def test_depth_to_space_gpu(self):
+        self._test_reorganize_data_gpu(tf.depth_to_space, [1, 4, 1, 1])
```
Comment (zhijxu-MS) [inline tests/test_backend.py]: add test  case where change is not h and w are not 1

**Comment by lei-Qiao** [comment]
I changed the shape in test case and faults in comments.

**Pull request merged**